platform/darwin/test/MGLOfflineStorageTests.mm

```diff
@@ -264,4 +264,49 @@ - (void)testResourceTransform {
     [os setDelegate:nil];
 }
 
+- (void)testAddingFileContent {
+    
+    NSURL *styleURL = [MGLStyle streetsStyleURLWithVersion:MGLStyleDefaultVersion];
+    
+    // Barcelona.
+    MGLCoordinateBounds bounds = {
+        { .latitude = 41.2724, .longitude = 2.0407 },
+        { .latitude = 41.4664, .longitude = 2.2680 },
+    };
+    double zoomLevel = 20;
+    MGLTilePyramidOfflineRegion *region = [[MGLTilePyramidOfflineRegion alloc] initWithStyleURL:styleURL bounds:bounds fromZoomLevel:zoomLevel toZoomLevel:zoomLevel];
+    
+    NSString *nameKey = @"Name";
+    NSString *name = @"Barcelona";
+    
+    NSData *context = [NSKeyedArchiver archivedDataWithRootObject:@{
+                                                                    nameKey: name,
+                                                                    }];
+    
+    [[MGLOfflineStorage sharedOfflineStorage] addPackForRegion:region withContext:context completionHandler:^(MGLOfflinePack * _Nullable completionHandlerPack, NSError * _Nullable error) {
```

Comment:
This test seems duplicative of existing tests of `-addPackForRegion:withContext:completionHandler:`.